packages/stv/src/index.test.ts

```diff
@@ -212,13 +212,48 @@ describe('calculateStvWinners', () => {
 
   it('should correctly redistribute votes when a candidate is eliminated', () => {
     const voteRecords: VoteRecord[] = [
-      { voteCount: 2, voteOrder: ['Alice', 'Charlie'] },
-      { voteCount: 1, voteOrder: ['Bob', 'Alice'] },
+      { voteCount: 1.2, voteOrder: ['Alice', 'Charlie'] },
+      { voteCount: 1.1, voteOrder: ['Bob', 'Charlie'] },
       { voteCount: 1, voteOrder: ['Charlie', 'Bob'] },
+      { voteCount: 0.9, voteOrder: ['Dave', 'Bob', 'Alice'] },
+    ];
+
+    const { winners, tieCount } = calculateStvWinners(voteRecords, 2);
+    expect(winners).toEqual(['Bob', 'Alice']); // Bob wins first, followed by Alice
+    expect(tieCount).toBe(0);
+  });
```

Comment:
Updated the test so it would actually trigger an elimination (I assume it did originally, but not in the updated code).